test/gtest/ucm/test_dlopen/dlopen_test_do_load.c

```diff
@@ -4,8 +4,11 @@
  * See file LICENSE for terms.
  */
 
+#include <ucs/sys/compiler.h>
+
 #include <dlfcn.h>
 
+UCS_F_NOOPTIMIZE /* prevent using tail recursion unwind */
```

Comment:
where is the tail recursion in this case? Can a `load_func` be `load_lib`?

Reply:
tail recursion here is **jump**(load_func) instead of **call**(load_func), that is why there is no stack frame to this function & loader failed to identify RPATH

load_func can't be load_lib